Description: Disable bluetooth by default
 blueman (2.0.8-1parrot1) rolling; urgency=medium
 .
   * Disable bluetooth by default.
Author: h0tw4t3r <h0tw4t3r@parrotsec.org>
Reviewed-By: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2019-08-20

--- blueman-2.0.8.orig/blueman/plugins/applet/PowerManager.py
+++ blueman-2.0.8/blueman/plugins/applet/PowerManager.py
@@ -25,7 +25,7 @@ class PowerManager(AppletPlugin):
     __options__ = {
         "auto-power-on": {
             "type": bool,
-            "default": True,
+            "default": False,
             "name": _("Auto power-on"),
             "desc": _("Automatically power on adapters")
         }
